driver-core/src/main/com/mongodb/internal/connection/DefaultServer.java

```diff
@@ -23,18 +23,19 @@
 import com.mongodb.MongoSecurityException;
 import com.mongodb.MongoSocketException;
 import com.mongodb.MongoSocketReadTimeoutException;
-import com.mongodb.internal.async.SingleResultCallback;
 import com.mongodb.connection.ClusterConnectionMode;
 import com.mongodb.connection.ServerDescription;
 import com.mongodb.connection.ServerId;
 import com.mongodb.connection.ServerType;
+import com.mongodb.connection.TopologyVersion;
```

Comment:
This file is where all the real functional changes are.  The rest is work that supports it.